application/betScanner.py

```diff
@@ -1,24 +1,24 @@
 import os
 from selenium import webdriver
 from application.parsers import TennisMarathonBetModule, Tennis1xbetModule
-import threading
+from concurrent.futures.thread import ThreadPoolExecutor
+import asyncio
 import logging
 import time
 
 
 def main():
     chrome_options = webdriver.ChromeOptions()
     chrome_options.binary_location = os.environ.get("GOOGLE_CHROME_BIN")
-    chrome_options.add_argument("--headless")
+    # chrome_options.add_argument("--headless")
     chrome_options.add_argument("--disable-dev-shm-usage")
     chrome_options.add_argument("--no-sandbox")
     chrome_options.add_argument("--log-level=3")
     driver_1xbet = webdriver.Chrome(executable_path=os.environ.get("CHROMEDRIVER_PATH"), options=chrome_options)
     driver_marathonbet = webdriver.Chrome(executable_path=os.environ.get("CHROMEDRIVER_PATH"), options=chrome_options)
-    threading.Thread(target=start_1xbet(driver_1xbet)).start()
-    threading.Thread(target=start_marathonbet(driver_marathonbet)).start()
```

Comment:
leave these code, we might need it. create two functions `run_in_parallel` and `run_asynchronously`